packages/react-catalog-view-extension/src/components/CatalogTile/CatalogTile.tsx

```diff
@@ -7,7 +7,6 @@ import { CardBody } from '@patternfly/react-core/dist/js/components/Card/CardBod
 import { CardFooter } from '@patternfly/react-core/dist/js/components/Card/CardFooter';
 import classNames from 'classnames';
 
-import { Omit } from '../../helpers/typeUtils';
 export interface CatalogTileProps extends Omit<React.HTMLProps<HTMLElement>, 'title'> {
```

Comment:
Do we really not need Omit anymore? It looks like we're still using it.

Reply:
Typescript 3.5 introduced the [new Omit helper type.](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-5.html#the-omit-helper-type)